### DISCUSSION TITLE
The last job (Provision branch) failed with an error

### DISCUSSION THREAD

**Discussion by teixemf** [netboxlabs/netbox-branching#93] [Q&A]
Hi, 
I upgraded netbox to v4.1 and I installed the netbox-branching plugin v0.4.0.
When I try to create a branch I get this error:

_The last job (Provision branch) failed with an error:_
_AttributeError("'NoneType' object has no attribute '_base_manager'")_

![image](https://github.com/user-attachments/assets/97e8fb04-43ad-4752-b1c7-a2796f876a2e)

Any idea of what  could be causing this error?
Thanks.

**Comment by sebastianreifetshammer** [comment]
same problem here, did you fix it somehow? :)

**Reply by JoAerts** [reply]
I'm having the same problem, one version 4.23 with the latest 0.53 branching release. Did you ever found a solution?
